create_dictionary.qmd

```diff
@@ -1,4 +1,11 @@
 # Creating a dictionary
 
-XXX
-a
+A core component of the traits.build database structure is a trait dictionary that clearly defines and describes each trait concept included within the trait database.
```

Comment:
`traits.build` rather than trait.build, throughout

Reply:
confused - it is traits.build here And I've searched the entire repository and they are all traits.build

Reply:
Sorry, my comment was suggesting to add the tick marks around traits.build, so "`traits.build`", not "traits.build"